views/Contacts/components/Locations/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,106 @@
+import { useCallback, useEffect, useState } from 'react';
+
+import { useAppDispatch, useAppSelector } from 'store';
+import {
+  selectCurrentLocation,
+  selectCurrentLocationIndex,
+  selectLocations,
+  setCurrentLocationIndex,
+} from 'store/mapSlice';
+
+import { Link, Text } from 'components';
+
+import styles from './Tabs.module.scss';
+
+const createTimeString = (timeZone: string) =>
+  new Intl.DateTimeFormat('en-GB', { timeStyle: 'short', timeZone }).format(new Date());
+
+export const Tabs = () => {
+  const locations = useAppSelector(selectLocations);
+  const currentLocation = useAppSelector(selectCurrentLocation);
+  const currentLocationIndex = useAppSelector(selectCurrentLocationIndex);
+  const dispatch = useAppDispatch();
+  const [currentLocationTime, setCurrentLocationTime] = useState('');
+
+  const updateTime = useCallback(
+    () => setCurrentLocationTime(createTimeString(currentLocation.timeZone)),
+    [currentLocation.timeZone],
+  );
+
+  useEffect(() => {
+    updateTime();
+
+    const interval = setInterval(updateTime, 1000);
+
+    return () => clearInterval(interval);
+  }, [updateTime]);
+
+  const handleTabClick = (index: number) => dispatch(setCurrentLocationIndex(index));
+
+  return (
+    <div className={styles.Tabs}>
+      <div className={styles.Tabs__TabsContainer}>
+        {locations.map((location, idx) => (
+          <button key={location.name} className={styles.Tabs__Button} onClick={() => handleTabClick(idx)}>
+            <Text
+              className={[
+                styles.Tabs__Button__Text,
+                currentLocationIndex === idx && styles['Tabs__Button__Text--Active'],
+              ]}
+              type="h5"
+              size="small"
+            >
+              {location.name}
+            </Text>
+          </button>
+        ))}
+      </div>
+
+      <div className={styles.Tabs__InfoPlace}>
```

Comment:
Move tab content to separate component (TabContent, for example), and use useAppSelector hook in for data rendering